src/components/profile/profile.tsx

```diff
@@ -67,13 +202,56 @@ const Profile = () => {
             .
           </p>
         </div>
-        <div style={{ margin: "1em 0" }}>
-          <Alert variant={user?.timeout ? "success" : "danger"}>
-            {user.timeout
-              ? "Your user session is active."
-              : "Your session has expired!"}
-          </Alert>
-        </div>
+        {userIVAs.length > 0
+          ? userIVAs.map((x, index) => {
+              return (
+                <Row key={x.id + index}>
+                  <Col xs={3}>
+                    {IVAType[x.type]}: {x.value}
+                  </Col>
+                  <Col xs={2}>
+                    {x.status === IVAStatus.Unverified ? (
+                      <Button
+                        id={"del_" + x.id}
+                        className="p-0 px-1 bg-quinary"
+                        onClick={(e) => {
+                          let button = e.target as HTMLButtonElement;
+                          HandleRequestVerification(button.id.substring(4));
+                        }}
+                      >
+                        Request verification
+                      </Button>
+                    ) : x.status === IVAStatus.CodeTransmitted ? (
+                      <Button
+                        id={"con_" + x.id}
+                        className="p-0 px-1 bg-quinary"
+                        onClick={(e) => {
+                          let button = e.target as HTMLButtonElement;
+                          HandleConfirmVerification(button.id.substring(4));
+                        }}
+                      >
+                        Enter verification code
+                      </Button>
+                    ) : (
+                      IVAStatus[x.status]
+                    )}
+                  </Col>
+                  <Col xs={1}>
+                    <Button
+                      variant="link"
+                      className="border-0 h-100 text-secondary p-0 d-flex align-items-center"
+                      onClick={() => {
+                        setToDeleteIVA(x);
+                        setShowDeletionConfirmationModal(true);
+                      }}
+                    >
+                      <FontAwesomeIcon icon={faCircleXmark} className="fa-lg" />
+                    </Button>
+                  </Col>
+                </Row>
+              );
+            })
+          : "You have not yet created any independent verification addresses. This is needed if you wish to download research data."}
```

Comment:
We should use *either* the term "verification address" *or* the term "contact address" on user facing pages. Currently it's a mix. I think both are fine, but it should be done consistently. The more technical term "IVA" should not be used on user facing pages.

Reply:
Done 👍